roles/confluent.kafka/templates/server_kerberos_ssl.properties.j2

```diff
@@ -1,7 +1,7 @@
 # Maintained by Ansible
-listeners=SASL_SSL://{{inventory_hostname}}:{{broker.config.sasl_port}},PLAINTEXT://{{inventory_hostname}}:9091
+listeners=SASL_SSL://{{inventory_hostname}}:{{kafka_port}},PLAINTEXT://{{inventory_hostname}}:9091
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.